shared/test/dataDriver_test.js

```diff
@@ -151,6 +151,19 @@ describe("loop.DataDriver", () => {
     });
   });
 
+  describe("#handleEvent.error", () => {
+    it("should retry the connection on error", () => {
+        let originalConnect = driver._connectToRoom;
+        sandbox.stub(driver, "_connectToRoom", token => {
+          originalConnect.call(driver, token);
+        });
+        driver.handleEvent({ type: "error" });
+        sinon.assert.notCalled(dispatcher.dispatch);
```

Comment:
Repeating comment from before: What action dispatch are we making sure isn't called?